lib/spack/spack/cmd/env.py

```diff
@@ -38,10 +38,16 @@
 
 
 def setup_parser(subparser):
+    subparser.epilog = 'To run a command with the emulated environment' \
+                       ' insert a \'--\' between the spec and the ' \
```

Comment:
Just wrap the string with " instead of ' and you won't need these backslashes.

Reply:
Oh boy, conflicting OCDs is problematic. Sorry for making you suffer! My logic is that Python allows 2 different string delimiters for a reason, and this is the reason! But OCD doesn't respond to logic well, so that might not help you...